### PR TITLE
Replace fat cli with a light one

### PR DESCRIPTION
**Pull request by diegosalvi**
This is a draft to create a slim cli while keeping "fat" cli artifact

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

### CODE REVIEW

**Review by eolivelli — CHANGES_REQUESTED** [review]
Makes sense to me

It is quite hard for the user to understand why have two CLI versions.

I guess that the problem is about the fact that by default from the CLI you can start the embedded server. 
I don't think that this is something that is expected.
I would simply fix the original CLI.
if you need to start a server over existing data you can download the zip distribution

**Comment by aluccaroni** [comment]
+1 to simplify the cli and keep only one, even if it's a breaking change

**Review by eolivelli — CHANGES_REQUESTED** [review]
As @aluccaroni said, it is better to have only one CLI tool and not two.
It would be confusing to have two packages

**Comment by diegosalvi** [comment]
Sounds good to me

**Comment by diegosalvi** [comment]
After tests this could be merged

File: herddb-cli/pom.xml [hunk 1]
```diff
@@ -59,31 +59,6 @@
     </dependencies>
     <build>
         <plugins>
-            <plugin>
```
Comment (eolivelli) [inline herddb-cli/pom.xml:62]: I think that if you drop this plugin the .jar won't be able to work.

We still need a uber jar, but we can remove anything that is not required to run the CLI.
I think that most of the weight is due to the rocksdb jar.
I would drop useless jars, like we did with the "thin" JDBC driver

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: herddb-cli/pom.xml (hunk 1 above)
Comment (diegosalvi) [reply herddb-cli/pom.xml:62]: The shaded jar generate various problems with his shaded dependencies.
You can just download the services and use the cli with it or in a more complex project run it with your other libraries.
Shading every dependency has many classloading problems when you want to upgrade some dependency (and then for some "strange" behavior you find loaded old classes from the shaded lib)

Comment (eolivelli) [reply herddb-cli/pom.xml:62]: The Uber jar is not intended to be used as dependency but only to run the CLI from the command line.

I agree that we can let users use the zip file.
Also, now that we have docker images, it is super easy to run docker run -it .....

**Review by eolivelli — COMMENTED** [review]
Do we have test Cades that very that the CLI packaged in the zip distro works?

Did you try it manually?

**Comment by diegosalvi** [comment]
Good catch! I forgot the cli shell and bat files in services.
Fixed services package:
* changed scripts to use java -cp instead of java -jar
* moved herddb-cli to lib instead of main directory (it's cleaner this way)

**Review by dmercuriali — APPROVED** [review]
Tested locally. LGTM

**Review by eolivelli — APPROVED** [review]
LGTM
nice work !

**Pull request merged**